### PR TITLE
Wireless requester terminal

### PR DESCRIPTION
**Pull request by Mari023**
add a wireless requester terminal
if ae2wtlib is installed, it integrates into the wireless universal terminal, and can be used with a quantum bridge card for unlimited range

things that still need to be considered:
- [x] the port to forge will not be quite straight forward, I can make a second PR for forge if you want
- [x] determine whether or not this should be ported to 1.18.2 (there would be no ae2wtlib integration in 1.18.2)
- [x] ~~Integration with MEGAs greater energy card, tho this is probably best done on MEGAs side (it should already work if both ae2wtlib and MEGA is installed due to MEGAs ae2wtlib integration)~~ MEGA will have to do this
- [ ] I should probably add some comments
- [ ] Some classnames could probably improved. I am bad at at naming things

I have ported this to forge 1.19.2 and fabric 1.20.1, I will do forge 1.20.1 once this is merged

### CODE REVIEW

**Comment by rlnt** [comment]
Thank you, I'll give it a look when I find the time.

A Forge PR would be appreciated but I don't think it will be that hard considering the project is split into a common code base and platform-based code inside the platform package.
I don't plan to backport this to 1.18.2. The mod is not used much on that version since they have LazierAE2. And I also do not plan to integrate MEGA. If they want to add integration, it should be on their side, as you said.

**Comment by Mithi83** [comment]
I'd like to revive this PR again. This feature is just what I was missing in my recent playthrough of FTB Skies.

File: src/main/resources/assets/merequester/lang/de_de.json
Comment (Mithi83) [inline src/main/resources/assets/merequester/lang/de_de.json:1]: Müsste „Open“ dann nicht auch „Öffne“ heißen oder ist das Absicht?

Comment (Mari023) [reply src/main/resources/assets/merequester/lang/de_de.json:1]: I just forgot to translate it

**Comment by TNoStone** [comment]
1.20.1 wireless requester terminal coming any time soon by chance?

**Comment by Mari023** [comment]
> 1.20.1 wireless requester terminal coming any time soon by chance?

no

**Comment by rlnt** [comment]
I don't think I am going to push any more feature updates to the ME Requester for older versions. And since the NeoForge transition is already ongoing and I don't have a lot of time, I will probably wait for 1.20.4/1.20.5/1.21 to implement this. AE2 itself is dropping Fabric so all add-on authors have to do that too. Even if some fork for Fabric happens, I will not support it I think. I am glad to return to a single loader again.

I think I'll close the PR and I'll use it as a reference for implementing support for the wireless terminal myself when doing the 1.20.4 NeoForge port.

Sorry for the inconvenience but it's not feasible right now.

**Pull request closed without merging**